charts/kkr/wallet-api/values.yaml

```
@@ -0,0 +1,108 @@
+# Default values for wallet-api.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: 293315485843.dkr.ecr.ap-northeast-2.amazonaws.com/kkr/wallet-api:0.0.3-96bbd7f-0
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
```

이 부분은 대체로 override 해서 사용하니 문제는 없는데, repository 와 tag 는 아래처럼 작성되는 게 맞으니 확인해주세요.
두 값을 `:` 으로 join 한 값이 이미지로 반영됩니다.

```suggestion
  repository: 293315485843.dkr.ecr.ap-northeast-2.amazonaws.com/kkr/wallet-api
  pullPolicy: IfNotPresent
  # Overrides the image tag whose default is the chart appVersion.
  tag: "0.0.3-96bbd7f-0"
```